services/121-service/src/migration/1724079447374-add-userId-to-twilio-message-entity.ts

```diff
@@ -0,0 +1,49 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUserIdToTwilioMessageEntity1724079447374
+  implements MigrationInterface
+{
+  name = 'AddUserIdToTwilioMessageEntity1724079447374';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."twilio_message" ADD "userId" integer`,
+    );
+
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."twilio_message" ALTER COLUMN "userId" SET NOT NULL`,
+    );
+
+    await queryRunner.query(`
+      UPDATE "121-service"."twilio_message" tm
+      SET "userId" = (
+        SELECT t."userId"
+        FROM "121-service"."transaction" t
+        WHERE t."id" = tm."transactionId"
+        LIMIT 1
+      )
+      WHERE tm."transactionId" IS NOT NULL
+    `);
+
+    await queryRunner.query(
+      `UPDATE "121-service"."twilio_message" SET "userId" = 1 WHERE "userId" IS NULL`,
```

Comment:
I tried to execute this migration against production data and Got that there is no user with `id = 1`, So I think its better to find the `userId` first by username and then user that userId.

![Screenshot 2024-08-20 at 12 11 50](https://github.com/user-attachments/assets/db98ca51-bf40-46ae-9392-da7d68fa3942)

Reply:
Done